### PR TITLE
Resolve not showing some icons in the icons page of the Playground project (#1862)

### LINKED ISSUE #1862: Some icons are not showing on the Icons page of the Playground project
Some of the icons on the icon page are missed. Also, some of the icons are not displayed correctly.
Here are some photos of these issues:

freesnippingtool com_capture_20220411180351
---
freesnippingtool com_capture_20220411180406

### PR DESCRIPTION
**Pull request by sajadkardel**
this closes #1862

### CODE REVIEW

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Bit.Client.Web.BlazorUI.Playground.Web.csproj [hunk 1]
```diff
@@ -34,4 +34,11 @@
         <ProjectReference Include="..\Shared\Bit.Client.Web.BlazorUI.Playground.Shared.csproj" />
     </ItemGroup>
 
+    <ItemGroup>
+      <Content Update="wwwroot\fonts\fabric-icons.woff">
+        <ExcludeFromSingleFile>true</ExcludeFromSingleFile>
+        <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+      </Content>
+    </ItemGroup>
+
```
Comment (maryamhdr) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Bit.Client.Web.BlazorUI.Playground.Web.csproj:43]: I think these changes must not be committed!

**Review by maryamhdr — CHANGES_REQUESTED** [review]

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Bit.Client.Web.BlazorUI.Playground.Web.csproj (hunk 1 above)
Comment (ysmoradi) [reply src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Bit.Client.Web.BlazorUI.Playground.Web.csproj:43]: @sajadkardel Do we really need this? @sajadkardel

Comment (msynk) [reply src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Bit.Client.Web.BlazorUI.Playground.Web.csproj:43]: no, these changes are redundant

File: src/Client/Web/Bit.Client.Web.BlazorUI/Styles/Fluent/fonts.fluent.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿@font-face {
     font-family: 'Fabric MDL2 Assets';
-    src: url('../assets/fluent/fonts/FabExMDL2.3.36.ttf') format("truetype");
+    src: url('../assets/fluent/fonts/fabric-icons.woff') format("woff");
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Styles/Fluent/fonts.fluent.scss:3]: add the version of the fabric icons to the file name.
the current version of the font is visible on the UI fabric font website (https://uifabricicons.azurewebsites.net/) as shown in the following image:
image

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Bit.Client.Web.BlazorUI.Playground.Web.csproj (hunk 1 above)
Comment (sajadkardel) [reply src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Bit.Client.Web.BlazorUI.Playground.Web.csproj:43]: No, I was wrong! I'm fixing it right now :) @ysmoradi

**Review by msynk — CHANGES_REQUESTED** [review]
beware of the huge changes.
bring back the old code style in both icons...scss files as shown in the following image:
image

File: src/Client/Web/Bit.Client.Web.BlazorUI/Styles/Fluent/fonts.fluent.scss (hunk 2 above)
Comment (msynk) [reply src/Client/Web/Bit.Client.Web.BlazorUI/Styles/Fluent/fonts.fluent.scss:3]: use a filename like this: `FabMDL2.v4.46.woff`

**Review by msynk — APPROVED** [review]

**Review by Cyrus-Sushiant — APPROVED** [review]

**Review by maryamhdr — APPROVED** [review]

**Pull request merged**